syz-ci/syz-ci.go

```diff
@@ -453,10 +441,19 @@ func loadManagerConfig(cfg *Config, mgr *ManagerConfig) error {
 	mgr.KernelBaselineConfig = osutil.Abs(mgr.KernelBaselineConfig)
 	mgr.KernelCmdline = osutil.Abs(mgr.KernelCmdline)
 	mgr.KernelSysctl = osutil.Abs(mgr.KernelSysctl)
-
 	if mgr.KernelConfig != "" && mgr.KernelBaselineConfig == "" {
 		mgr.KernelBaselineConfig = inferBaselineConfig(mgr.KernelConfig)
 	}
+	if err := mgr.validate(cfg); err != nil {
+		return err
+	}
+
+	if cfg.PatchVMConfigs[managercfg.Type] != nil {
+		managercfg.VM, err = config.MergeJSONs(managercfg.VM, cfg.PatchVMConfigs[managercfg.Type])
```

Comment:
Can this cause a situation when we add serial_port_user to qemu config and then it will fail to parse due to unknown field?